src/snowflake/snowpark/session.py

```diff
@@ -1865,7 +1865,7 @@ def write_pandas(
     def create_dataframe(
         self,
         data: Union[List, Tuple, "pandas.DataFrame"],
-        schema: Optional[Union[StructType, List[str]]] = None,
+        schema: Optional[Union[StructType, List[str], Tuple[str]]] = None,
```

Comment:
Should we make sure we use `schema` as an iterable when it is not `StructType`, and then support `Iterable[str]`?